[msx] test Started org.apache.hadoop.hdfs.server.datanode.TestDataNodeVolumeFailureToleration#testFailedVolumeOnStartupIsCounted
[msx] unitTestCounterInClass = 0
2020-03-14 07:42:29,428 [Thread-0] INFO  namenode.NameNode (NameNode.java:<clinit>(931)) - [msx-restart] reconfPoint = 0
2020-03-14 07:42:30,102 [Thread-0] INFO  namenode.NameNode (NameNode.java:performReconfig(958)) - [msx-restart] NameNode init, reconfigMode is none, do nothing
2020-03-14 07:42:31,046 [Thread-0] INFO  namenode.NameNode (NameNode.java:<init>(1042)) - [msx-restart] NameNode 36686 start
2020-03-14 07:42:31,063 [Thread-0] INFO  datanode.DataNode (DataNode.java:<clinit>(433)) - [msx-restart] reconfPoint = 0
2020-03-14 07:42:31,140 [Thread-0] INFO  datanode.DataNode (DataNode.java:performReconfig(460)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2020-03-14 07:42:31,448 [Thread-0] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2750)) - [msx-restart] DataNode 42287 start
2020-03-14 07:42:32,140 [Thread-0] INFO  datanode.DataNode (DataNode.java:shutdown(2075)) - [msx-restart] DataNode 42287 stop, double check before stop, dfs.data.transfer.client.tcpnodelay = true
2020-03-14 07:42:32,196 [Thread-0] INFO  datanode.DataNode (DataNode.java:performReconfig(460)) - [msx-restart] DataNode init, reconfigMode is none, do nothing
2020-03-14 07:42:32,237 [Thread-0] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2750)) - [msx-restart] DataNode 42184 start
2020-03-14 07:43:02,356 [Thread-0] INFO  datanode.DataNode (DataNode.java:shutdown(2075)) - [msx-restart] DataNode 42184 stop, double check before stop, dfs.data.transfer.client.tcpnodelay = true
2020-03-14 07:43:02,510 [Thread-0] INFO  namenode.NameNode (NameNode.java:stop(1087)) - [msx-restart] NameNode 36686 stop, double check before stop, dfs.data.transfer.client.tcpnodelay = true
[msx] test Failure org.apache.hadoop.hdfs.server.datanode.TestDataNodeVolumeFailureToleration#testFailedVolumeOnStartupIsCounted
[msx] writeFile testName = org.apache.hadoop.hdfs.server.datanode.TestDataNodeVolumeFailureToleration#testFailedVolumeOnStartupIsCounted
[msx] failed
[msx] failureMessage: Timed out waiting for capacity. Live = 1 Expected = 1 Dead = 0 Expected = 0 Total capacity = 49033805824 Expected = 24516902912 Vol Fails = 0 Expected = 1
[msx] stackTrace: java.util.concurrent.TimeoutException: Timed out waiting for capacity. Live = 1 Expected = 1 Dead = 0 Expected = 0 Total capacity = 49033805824 Expected = 24516902912 Vol Fails = 0 Expected = 1
2020-03-14 07:43:02,596 [main] INFO  server.JournalNode (JournalNode.java:<clinit>(168)) - [msx-restart] reconfPoint = 0
[msx] reset performed
[msx] test Finished org.apache.hadoop.hdfs.server.datanode.TestDataNodeVolumeFailureToleration#testFailedVolumeOnStartupIsCounted
[msx] writeFile testName = org.apache.hadoop.hdfs.server.datanode.TestDataNodeVolumeFailureToleration#testFailedVolumeOnStartupIsCounted
[msx] Info : file existed /root/parameter_test_controller/shared/test_results/org.apache.hadoop.hdfs.server.datanode.TestDataNodeVolumeFailureToleration#testFailedVolumeOnStartupIsCounted
[msx] succeed
[msx] reset performed
[msx] all testRunFinished
